src/components/content/deployServiceConfigParameter/DeploymentServiceConfigurationParameter.tsx

```diff
@@ -0,0 +1,233 @@
+                        }
+                    },
+                },
+            ],
+        },
+        {
+            title: <div className={deploymentVariablesStyles.variablesColumns}>ModificationImpact</div>,
+            dataIndex: 'modificationImpact',
+            children: [
+                {
+                    title: <div className={deploymentVariablesStyles.variablesTableChildrenTitle}>Is Data Lost</div>,
+                    dataIndex: ['modificationImpact', 'isDataLost'],
+                    key: 'isDataLost',
+                    render: (text: boolean | undefined | null) => {
+                        if (text === true) {
+                            return <div className={deploymentVariablesStyles.variablesColumns}>{`true`}</div>;
+                        } else if (text === false) {
+                            return <div className={deploymentVariablesStyles.variablesColumns}>{`false`}</div>;
+                        } else {
+                            return null;
+                        }
+                    },
+                },
+                {
+                    title: (
+                        <div className={deploymentVariablesStyles.variablesTableChildrenTitle}>
+                            Is Service Interrupted
+                        </div>
+                    ),
+                    dataIndex: ['modificationImpact', 'isServiceInterrupted'],
+                    key: 'isServiceInterrupted',
+                    render: (text: boolean | undefined | null) => {
+                        if (text === true) {
+                            return <div className={deploymentVariablesStyles.variablesColumns}>{`true`}</div>;
+                        } else if (text === false) {
+                            return <div className={deploymentVariablesStyles.variablesColumns}>{`false`}</div>;
+                        } else {
+                            return null;
+                        }
+                    },
+                },
+            ],
+        },
+        {
+            title: <div className={deploymentVariablesStyles.variablesColumns}>IsReadOnly</div>,
+            dataIndex: 'isReadOnly',
+            render: (text: string) => {
+                return <div className={deploymentVariablesStyles.variablesColumns}>{text}</div>;
+            },
+        },
+        {
+            title: <div className={deploymentVariablesStyles.variablesColumns}>ManagedBy</div>,
+            dataIndex: 'managedBy',
+            render: (text: string) => {
+                return <div className={deploymentVariablesStyles.variablesColumns}>{text}</div>;
+            },
+        },
+    ];
+
+    return (
+        <>
+            <h3 className={catalogStyles.catalogDetailsH3}>
+                <FileTextOutlined />
+                &nbsp;Service Configuration Parameters
+            </h3>
+            <div className={deploymentVariablesStyles.variablesTableContainer}>
+                <Table
```

Comment:
is it possible to fit the table within the space available without any scrollbar? Looks a bit odd with the scroll bar. Please see if antd component provides a way to configure this.

Reply:
is this solved?